chainer_chemistry/dataset/parsers/csv_file_parser.py

```diff
@@ -160,24 +161,12 @@ def parse(self, filepath, retain_smiles=False):
             # Spec not finalized yet for general case
             result = pp.process(filepath)
 
+        smiles_list = numpy.array(smiles_list) if return_smiles else None
```

Comment:
I feel weired to name `smiles_list` even when we make it to numpy array.
could you change it to 

```smiles = numpy.array(smiles_list) if return_smiles else None```

or `smiles_array` ?

Reply:
I'm going to use `smileses`.